### ISSUE TITLE
Token in MultiInput `AND` or `OR`?

### ISSUE THREAD

**Issue by marianfoo** [ui5-community/bestofui5-website#134]
Currently if you enter multiple tokens, they filter with `OR`,
so if you use type "middleware" and "tag" wdi5", all packages with both are displayed.

So: Use here  `AND` or `OR` ?
![image](https://user-images.githubusercontent.com/13335743/164388042-68d5834c-f05e-4f19-96d2-49bbf304713e.png)

**Comment by wridgeu** [comment]
Imho. I would use the search/multiinput to narrow down my selection or rather the result set, so `AND` would feel a bit more natural to me. But I guess thats just my personal preference and I'm by no means an UX designer. When using `OR` there is a certain threshold of tokens that "makes sense" until everything will be displayed in the end anyway which defeats it's purpose a bit, I guess?

It also depends on the amount of actual use cases where multiple "tokens" happen to match one entry. If there is always an either/or case, switching to `AND` would result in a worse UX than now as you're more likely to receive nothing as result. But this could be intentional and very well what we want to happen as there is just nothing matching the selection.

For example using `AND` for the `css` and `module` would then result in nothing. Doesn't have to be a bad thing, it's just a ... choice 😅
![image](https://user-images.githubusercontent.com/14982812/164500008-0347f01c-a678-4c09-9db7-6856ee68c536.png)

TL;DR - `AND` for me. 😊

**Comment by marianfoo** [comment]
Thank you, I agree with your comments.
It's nice that someone also reads the issues and that I'm not just talking to myself here 😃
I created the ticket because I instinctively searched with AND and then realised that it was better this way.
After all, you search precisely to find something specific.
Then I'll change it to AND

**Comment by wridgeu** [comment]
I actually also recently used it and wondered "oh it's OR, not quite what I thought but aight" 😅

> It's nice that someone also reads the issues and that I'm not just talking to myself here 😃

No worries I'm lurking actively 👀😆
